README.md

```diff
@@ -76,72 +72,89 @@ Installation
      After that, you must reboot your system to be sure everything works
      properly (libvirt iptables rules, ip forwarding, etc.).
 
-Running
--------
+## Running
 
-For a complete list of tasks, run:
+This is the workflow used to run integration tests:
 
-    $ rake -T
+* Create an ISO image to test. This ISO will contain the latest YaST packages,
+  although it's also possible to use any other image.
+* Install the system using the generated ISO and run the tests on the resulting
+  system. You can run the tests as many times as you want using the same
+  installed system.
 
-To run the testsuite, use the `test` Rake task:
+Actually, the second step can be seen as two different steps (building the
+system and running the tests). But for now, we'll keep them tied.
 
-    $ rake test
+After this brief introduction, let's go deeper into each step.
 
-This runs all tests defined in `test` directory (e.g. `test/tftp.rb`):
-* Building a KVM image by using the AutoYaST configuration file (e.g. `tftp.xml`)
-  You can watch the installation by using `virt-manager`. The image is `autoyast`.
-* Starting the built image.
-  You can watch it by using `virt-manager`. The image is `vagrant_autoyast_vm`.
-* Running RSpec tests on this machine which are defined in e.g. `test/tftp.rb`.
+### Generating an ISO
 
-To run only one single test use:
+Generating a new ISO to use in tests is as easy as typing:
 
-    $ rake test[<path_to_test_file>]
+    $ rake build_iso[sles12-sp1]
 
-e.g. `rake test[test/sles12.rb]`
+where `sles12-sp1` could be any of the values defined in
+[config/definitions.yml](https://github.com/yast/autoyast-integration-test/blob/master/config/definitions.yml).
+At this time SLE 12 (`sles12`) and SLE 12 SP1 (`sles12-sp1`) are supported.
 
-or you can also run any single script directly
+These are the steps that will be performed by this task:
 
-    $ bundle exec rspec [<path_to_test_file>]
+* If the original ISO is not found in the `iso` directory, it will be downloaded.
+* Latest YaST packages will also be downloaded from build system (IBS/OBS). If you
+  want to include your own packages, just drop them into `rpms/<definition>` directory
+  (e.g. `rpms/sles12-sp1`).
+* Those packages will be included in a DUD.
```

Comment:
This is the first mention about DUD, so better writer Driver Update (DUD) and reference that part in Linuxrc#p_dud wiki page.